cmd/kubeadm/app/cmd/reset.go

```diff
@@ -160,6 +164,10 @@ func AddResetFlags(flagSet *flag.FlagSet, resetOptions *resetOptions) {
 		&resetOptions.dryRun, options.DryRun, resetOptions.dryRun,
 		"Don't apply any changes; just output what would be done.",
 	)
+	flagSet.BoolVar(
+		&resetOptions.cleanupTmpDir, options.CleanupTmpDir, resetOptions.cleanupTmpDir,
+		"Indicate whether reset will cleanup the temp dir",
```

Comment:
`("Cleanup the %q directory", path.Join(kubeadmconstants.KubernetesDir, kubeadmconstants.TempDirForKubeadm))`

to show the user what we are cleaning

Reply:
Done